bin/watch-packages-and-rebuild.js

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+const fs = require( 'fs' );
+const path = require( 'path' );
+const { spawnSync } = require( 'child_process' );
+const chokidar = require( 'chokidar' );
+
+const packagesDirectoryPath = path.join( '.', 'packages' );
+const watcher = chokidar.watch( packagesDirectoryPath, {
+	ignored: /^\./,
+	persistent: true,
+	ignoreInitial: true,
+} );
+watcher.on( 'change', handleChange ).on( 'add', handleChange ).on( 'unlink', handleChange );
+
+let rebuildQueue = [];
+
+setTimeout( processRebuildQueue, 100 );
+
+function processRebuildQueue() {
+	rebuildQueue.map( rebuildPackage );
```

Comment:
nit: should be `.forEach()` instead of `.map()`, as we are not returning a new array
```suggestion
	rebuildQueue.forEach( rebuildPackage );
```